store/setup/utils.js

```diff
@@ -0,0 +1,37 @@
+import { ethers } from 'ethers';
+import { formatToEth } from 'common-util/functions';
+
+export const getNextReleasableAmount = (lockedBalance, timestamp) => {
+  const STEP_TIME = 365 * 86400;
+  let buolasNextReleasableTime = null;
+  let buolasNextReleasableAmount = null;
+
+  const startTime = Number(lockedBalance.startTime);
+  const endTime = Number(lockedBalance.endTime);
+
+  if (endTime === 0) {
+    // Revoke has been applied
+    buolasNextReleasableTime = new Date();
+    buolasNextReleasableAmount = 0;
+  } else {
```

Comment:
I believe this can be substituted with just a single condition:
`if (endTime > 0) {...}`
since if there is no next releasable amount, there's no need to output the next releasable date. The actual releasable amount is either there, or zero as well.